src/data_access/FileFolderDataAccessObject.java

```diff
@@ -128,7 +129,31 @@ public Folder get(String folderName) {
      * @param foldername
      * @return
      */
-    public HashMap<Integer, Recipe> getrecipeMap(String foldername) {
-        return folders.get(foldername).getRecipeMap();
+  // TODO change it to integer list<recipe information> use java doc to explain
+//    list[url, title]
+    public HashMap<Integer, ArrayList> getrecipeMap(String foldername) {
```

Comment:
I changed this function to the newest version

Reply:
did you push that to the main? if not, use mine for now and you can merge your things to the main.